docs/data_ingestion.md

```diff
@@ -1,21 +1,24 @@
-# Indexing documents for the Chat App
+# RAG chat: Data ingestion
```

Comment:
I didn't put the frontmatter on this one, as it feels the least like it merits its own page in the samples browser.